src/main/scala/IntegrationTests.scala

```diff
@@ -0,0 +1,36 @@
+package tests
+
+import org.apache.spark.SparkContext
+import pipelines.images.cifar.RandomPatchCifar
+import pipelines.images.cifar.RandomPatchCifar.RandomCifarConfig
+import pipelines.images.voc.VOCSIFTFisher
+import pipelines.images.voc.VOCSIFTFisher.SIFTFisherConfig
+import pipelines.text.NewsgroupsPipeline
+import pipelines.text.NewsgroupsPipeline.NewsgroupsConfig
+
+object IntegrationTests {
+  val cifar = RandomPatchCifar.run(_: SparkContext,
+    RandomCifarConfig(
+      trainLocation="./cifar_train.bin",
```

Comment:
I'll have the path be an environment variable that is used when doing the copy and when reading stuff, and error out if its unset.